OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -49,13 +50,15 @@ public Actor[] ActorsInBox(WPos topLeft, WPos bottomRight, LuaFunction filter =
 			return FilteredObjects(actors, filter).ToArray();
 		}
 
-		// HACK: This api method abuses the coordinate system, and should be removed
+		// HACK: This API method abuses the coordinate system, and should be removed
 		// in favour of proper actor queries.  See #8549.
+		[Obsolete("This function will be removed in favour of proper actor queries.")]
 		[Desc("Returns the location of the top-left corner of the map (assuming zero terrain height).")]
 		public WPos TopLeft => Context.World.Map.ProjectedTopLeft;
```

Comment:
This is still used by https://github.com/OpenRA/OpenRA/blob/6a31b1f9f368109cc1916ba6513f4c0eb30cf8b6/mods/ra/maps/soviet-05/soviet05.lua#L11 which should be changed as well then.